src/controls/extras/treeitem.coffee

```diff
@@ -34,16 +34,25 @@ class TreeItem #extends Base
   handle_relevant_or_required: ->
     tree_div = @tree.tree_root
     current_node = tree_div.find("[node_value = '" + @value + "']")
-
-    current_node.attr('relevant', @node_relevant())
+    current_node.attr('item-required', @node_required())
```

Comment:
ok, I guess that makes sense.

I have updated the spec tests per your suggestions